nodejs/week1/avg.js

```diff
@@ -0,0 +1,21 @@
+const input = process.argv.slice(2);
+
+const getAvg = (arguments) => {
```

Comment:
'arguments' could be a reserved javascript variable name. Please try to change it to something else
https://www.w3schools.com/js/js_reserved.asp